docs/cost/index.md

```diff
@@ -89,12 +80,112 @@ The following mapping tables describe:
 ```eval_rst
 .. csv-table-no-translate::
    :header-rows: 1
-   :widths: 20 20 30 30
+   :widths: 20 20 20 30
    :file: ../../build/current_lang/process-level-procurement.csv
 ```
 
 #### Implementation
 
+```eval_rst
+.. csv-table-no-translate::
+   :header-rows: 1
+   :widths: 20 20 20 30
+   :file: ../../build/current_lang/process-level-implementation.csv
+```
+
+## OCDS to OC4IDS Mapping
+
+### Guidance
+
+#### Command-line tool and reference implementation
+
+OCDS Kit's [convert-to-oc4ids](https://ocdskit.readthedocs.io/en/latest/cli/ocds.html#convert-to-oc4ids) command  is a command-line tool and reference implementation for converting OCDS data to OC4IDS format.
+
+`convert-to-oc4ids` covers most mappings in the following categories:
+
+* project-level identification
+* project-level preparation
+* process-level procurement
+
+However, `convert-to-oc4ids` does not cover all mappings, nor does it perform currency conversions. Mappings which `convert-to-oc4ids` does not cover are shown in *italics*.
+
+#### Mapping codelists
+
+Mappings that depend the specific classifications or codelists used in the OCDS data are not documented in detail, as they may differ by publisher. For example, mapping to the OC4IDS projectSector codelist.
+
+#### Alternative mappings
+
+Some mappings offer optional alternatives in case the primary mapping isn't available. For example, for OCDS data in which `planning.project.title` isn't available, you can optionally set the project `title` based on the `tender.title`.
+
+In order to provide analysts with additional context, alternative mappings may copy additional fields which don't appear in OC4IDS schema. You should remove these fields if you plan to publish your OC4IDS data.
+
+#### OCDS Extensions
+
+Some mappings use fields from [OCDS extensions](https://standard.open-contracting.org/latest/en/guidance/map/extensions/#extensions). In these cases, the names of extensions are noted in parentheses; where possible, alternative mappings are provided that use only fields from the core OCDS schema.
+
+#### Handling conflicts and duplicates
```

Comment:
I think some of these limitations should instead be documented in OCDS Kit. Otherwise, if we fix some limitations, or make OCDS Kit warn or error when there are conflicts, then we will have to edit this documentation. It'd be simpler to document this in proximity to the tool.

Reply:
I'm not sure I followed. This section describes some possible attributes of OCDS data that would require extra attention from anyone implementing the mapping in their own software. OCDS Kit's approach to these challenges isn't described here, rather there is a link to the OCDS Kit documentation.

Reply:
Ah, okay, fine to leave here then.